aospy/calc.py

```diff
@@ -767,3 +767,19 @@ def load(self, dtype_out_time, dtype_out_vert=False, region=False,
         if plot_units:
             data = self.var.to_plot_units(data, dtype_vert=dtype_out_vert)
         return data
+
+    def addAttrs(self, data):
```

Comment:
Note that PEP8 [suggests](https://www.python.org/dev/peps/pep-0008/#function-names) using function names that are all lowercase, with words separated by underscores.  For this function I would recommend using something more descriptive like `add_units_and_description_attrs`.